CallsheetLocalizations/it.xcloc/Localized Contents/it.xliff

```diff
@@ -278,151 +325,188 @@
         <note/>
       </trans-unit>
       <trans-unit id="Data provided by:" xml:space="preserve">
         <source>Data provided by:</source>
+        <target state="translated">Informazioni fornite da:</target>
         <note/>
       </trans-unit>
       <trans-unit id="Default Spoiler Settings" xml:space="preserve">
         <source>Default Spoiler Settings</source>
+        <target state="translated">Impostazioni Spoiler di default:</target>
         <note/>
       </trans-unit>
       <trans-unit id="Delete recent searches" xml:space="preserve">
         <source>Delete recent searches</source>
+        <target state="translated">Cancella ricerche recenti</target>
         <note/>
       </trans-unit>
       <trans-unit id="Delete search history" xml:space="preserve">
         <source>Delete search history</source>
+        <target state="translated">Cancella cronologia ricerche</target>
         <note/>
       </trans-unit>
       <trans-unit id="Details" xml:space="preserve">
         <source>Details</source>
+        <target state="translated">Dettagli</target>
         <note/>
       </trans-unit>
       <trans-unit id="Discover" xml:space="preserve">
         <source>Discover</source>
+        <target state="translated">Scopri</target>
         <note/>
       </trans-unit>
       <trans-unit id="Done" xml:space="preserve">
         <source>Done</source>
+        <target state="translated">Fatto</target>
         <note/>
       </trans-unit>
       <trans-unit id="Each TV show can have their own settings, set by tapping the `Hide Spoilers…` button when looking at a show or episode." xml:space="preserve">
         <source>Each TV show can have their own settings, set by tapping the `Hide Spoilers…` button when looking at a show or episode.</source>
+        <target state="translated">Ogni Serie TV può avere le proprie impostazioni, settate toccando il bottone `Nascondi Spoiler` quando guardi uno show od episodio.</target>
         <note/>
       </trans-unit>
       <trans-unit id="Ends at %@" xml:space="preserve">
         <source>Ends at %@</source>
+        <target state="translated">Finisce alle %@</target>
         <note>When a show or movie ends if you start it right now</note>
       </trans-unit>
       <trans-unit id="Episode" xml:space="preserve">
         <source>Episode</source>
+        <target state="translated">Episodio</target>
         <note/>
       </trans-unit>
       <trans-unit id="Episode Poster" xml:space="preserve">
         <source>Episode Poster</source>
+        <target state="translated">Poster Episodio</target>
         <note/>
       </trans-unit>
       <trans-unit id="Episode Scores" xml:space="preserve">
         <source>Episode Scores</source>
+        <target state="translated">Punteggio episodio [tricky one in Italian, I assumed you meant rating, but it could also translate as Colonna sonora episodio, if it refers to the soundtrack]</target>
```

Comment:
Yes, this means rating from 0-100. I can see how that would be ambiguous though!